pandora-client-web/src/graphics/def.ts

```diff
@@ -56,14 +71,20 @@ export const PRIORITY_ORDER_SPRITES = [
 	'BELOW_BREASTS',
 	'BREASTS',
 	'ABOVE_BODY',
+	'BELOW_ARM_LEFT_BACK',
+	'BELOW_ARM_LEFT_FRONT',
+	'ARM_LEFT_BACK',
+	'ARM_LEFT_FRONT',
+	'ABOVE_ARM_LEFT_BACK',
+	'ABOVE_ARM_LEFT_FRONT',
+	'BELOW_ARM_RIGHT_BACK',
+	'BELOW_ARM_RIGHT_FRONT',
+	'ARM_RIGHT_BACK',
+	'ARM_RIGHT_FRONT',
+	'ABOVE_ARM_RIGHT_BACK',
+	'ABOVE_ARM_RIGHT_FRONT',
 	'FRONT_HAIR',
 	'ABOVE_FRONT_HAIR',
```

Comment:
is this intentional, to put these above the arms?

(wouldn't ask if it were a separate commit)

Reply:
Yes, done to bring this layering more closer to the normal layering. By this point I'm thinking of just dropping the sprite order, as it doesn't really help anything anymore. Its original purpose was simplified layering when sprites didn't properly interleave in old demos, which I think changed with react-pixi rewrite.